zebra-chain/src/orchard/tests/prop.rs

```diff
@@ -0,0 +1,35 @@
+use proptest::prelude::*;
+use std::io::Cursor;
+
+use crate::{
+    orchard,
+    serialization::{ZcashDeserializeInto, ZcashSerialize},
+};
+
+proptest! {
+    #[test]
+    // Make sure only valid flags deserialize
```

Comment:
Nitpick: We might as well use a doc-comment here, rather than a standard comment.

And we usually put doc comments before attributes like `#[test]`.
```suggestion
    /// Make sure only valid flags deserialize
    #[test]
```